books/free-programming-books-zh.md

```diff
@@ -379,7 +371,7 @@
 
 ### Elixir
 
-* [Elixir 编程语言教程](https://elixirschool.com/cn/) (Elixir School)
+* [Elixir 编程语言教程](https://web.archive.org/web/20200224214134/http://elixirschool.com/blog) (Elixir School) *(:card_file_box: archived)*
```

Comment:
Splitted into HANS and HANT. Use one of:
- https://elixirschool.com/zh-hans
- https://elixirschool.com/zh-hant